osu.Game/Overlays/WikiOverlay.cs

```diff
@@ -115,20 +133,7 @@ private void onSuccess(APIWikiPage response)
             }
             else
             {
-                LoadDisplay(new WikiMarkdownContainer
-                {
-                    RelativeSizeAxes = Axes.X,
-                    AutoSizeAxes = Axes.Y,
-                    CurrentPath = $@"{api.WebsiteRootUrl}/wiki/{path.Value}/",
-                    Text = response.Markdown,
-                    DocumentMargin = new MarginPadding(0),
-                    DocumentPadding = new MarginPadding
-                    {
-                        Vertical = 20,
-                        Left = 30,
-                        Right = 50,
-                    },
-                });
+                LoadDisplay(articlePage = new WikiArticlePage($@"{api.WebsiteRootUrl}/wiki/{path.Value}/", response.Markdown));
```

Comment:
Could also use `WikiArticlePage` in the `onFail()` scenario to decrease layout code duplication, I think?

Reply:
Yup, it could do.